_docs/aws_resources.md

```diff
@@ -7,3 +7,18 @@ permalink: /aws_resources.html
 ---
 
 # AWS Resources, SDK Links, Tutorials, and Helpful Tidbits
+
+## Avoid Charges In The Free Tier
+
+After you have signed up for an AWS account and begin using services, keep in mind a few things to avoid unnecessary charges from using services that are not covered within Free Tier:
+
+1. Make sure services that you are using are covered by Free Tier. The list of those services can be found [here](https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc).
+
+2. [Set up Free Tier usage alerts](https://console.aws.amazon.com/billing/home#/preferences). Remember to keep an eye on your email to see if there is any usage alert from AWS.
+
+3. Monitor your costs and usage in [Billing and Cost Management Console](https://console.aws.amazon.com/billing).
+
+4. Set up a cost budget. [Click here for more information](https://aws.amazon.com/getting-started/tutorials/control-your-costs-free-tier-budgets/).
+
+5. Clean up test resources when you're done using them. See the [AWS documentation](https://aws.amazon.com/documentation/) for information about how to clean up resources after you're done using.
```

Comment:
This is a bit too vague for my liking; I understand the intention behind this, but it would be good to specifically mention commonly used products which, if left running, could incur high costs. EC2, DynamoDB, API Gateway, CloudWatch, and S3 all come to mind.

Consider adding an indented bulleted section for each common product; suggestion is below.

```suggestion
5. Clean up test resources when you're done using them. See the AWS documentation for each product in use for information about how to clean up resources after you're done using. Links for deleting or de-allocating a few common products are listed here.
  * [Delete S3 buckets.](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/delete-bucket.html)
  * [Terminate EC2 instances -- leaving EC2 instances running is a common cause of unexpected recurring charges.](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/terminating-instances.html)
  * Delete database tables: [DynamoDB](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/getting-started-step-8.html), [RDS](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_DeleteInstance.html), and/or [Redshift](https://docs.aws.amazon.com/redshift/latest/mgmt/managing-clusters-console.html#delete-cluster)
```